packages/block-editor/src/hooks/dimensions.js

```diff
@@ -77,6 +77,7 @@ export function DimensionsPanel( props ) {
 			<InspectorControls __experimentalGroup="dimensions">
 				{ ! isPaddingDisabled && (
 					<ToolsPanelItem
+						className="tools-panel-item-spacing"
```

Comment:
Styling nit: I think the styling changes here affect the position of the Unlink button in the `BoxControl` layout when the spacing presets are set to `0` steps:

| Trunk | This PR |
| --- | --- |
| <img width="285" alt="image" src="https://user-images.githubusercontent.com/14988353/184295856-717d87d4-15ec-4ca4-978d-ac339d5ef7ca.png"> | <img width="275" alt="image" src="https://user-images.githubusercontent.com/14988353/184296024-097af4a4-ad8f-484b-b6af-fe56ada774e4.png"> |

Since by default the spacing presets will be available, this feels like an edge case / bit of tweaking we could look into in a follow up to me.